examples/EventV1/EventV2.json

```diff
@@ -0,0 +1,26 @@
,8,ANALYTE1,ANL1,195.208,,,Y,1,Y,1\nSTUDY123,STUDY123-101-8,8,101,26,M,110,188,Treatment 2,VISIT 1,1,12,11.4,11.4,ANALYTE1,ANL1,134.832,,,Y,1,Y,1\nSTUDY123,STUDY123-101-8,8,101,26,M,110,188,Treatment 2,VISIT 2,2,0,0,0,ANALYTE1,ANL1,0,,,Y,1,Y,1\nSTUDY123,STUDY123-101-8,8,101,26,M,110,188,Treatment 2,VISIT 2,2,0.5,0.47,0.47,ANALYTE1,ANL1,34.79,,,Y,1,Y,1\nSTUDY123,STUDY123-101-8,8,101,26,M,110,188,Treatment 2,VISIT 2,2,1,1,1,ANALYTE1,ANL1,,Y,BAD LUCK,,,,\nSTUDY123,STUDY123-101-8,8,101,26,M,110,188,Treatment 2,VISIT 2,2,1.5,1.5,1.5,ANALYTE1,ANL1,143.336,,,Y,1,Y,1\nSTUDY123,STUDY123-101-8,8,101,26,M,110,188,Treatment 2,VISIT 2,2,2,2,2,ANALYTE1,ANL1,131.305,,,Y,1,Y,1\nSTUDY123,STUDY123-101-8,8,101,26,M,110,188,Treatment 2,VISIT 2,2,3,3,3,ANALYTE1,ANL1,119.275,,,Y,1,Y,1\nSTUDY123,STUDY123-101-8,8,101,26,M,110,188,Treatment 2,VISIT 2,2,5,5,5,ANALYTE1,ANL1,119.045,,,Y,1,Y,1\nSTUDY123,STUDY123-101-8,8,101,26,M,110,188,Treatment 2,VISIT 2,2,8,8,8,ANALYTE1,ANL1,98.611,,,Y,1,Y,1\nSTUDY123,STUDY123-101-8,8,101,26,M,110,188,Treatment 2,VISIT 2,2,12,12,12,ANALYTE1,ANL1,68.111,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 1,1,0,0,0,ANALYTE1,ANL1,0,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 1,1,0.5,0.5,0.5,ANALYTE1,ANL1,215.068,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 1,1,1,1,1,ANALYTE1,ANL1,441.704,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 1,1,1.5,1.38,1.38,ANALYTE1,ANL1,461.528,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 1,1,2,2,2,ANALYTE1,ANL1,426.557,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 1,1,3,3,3,ANALYTE1,ANL1,427.777,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 1,1,5,5,5,ANALYTE1,ANL1,324.493,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 1,1,8,8,8,ANALYTE1,ANL1,,Y,BAD LUCK,,,,\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 1,1,12,12,12,ANALYTE1,ANL1,268.174,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 2,2,0,0,0,ANALYTE1,ANL1,0,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 2,2,0.5,0.5,0.5,ANALYTE1,ANL1,249.804,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 2,2,1,0.9,0.9,ANALYTE1,ANL1,513.044,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 2,2,1.5,1.5,1.5,ANALYTE1,ANL1,536.069,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 2,2,2,2,2,ANALYTE1,ANL1,495.451,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 2,2,3,2.71,2.71,ANALYTE1,ANL1,,Y,BAD LUCK,,,,\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 2,2,5,4.52,4.52,ANALYTE1,ANL1,376.902,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 2,2,8,8,8,ANALYTE1,ANL1,437.357,,,Y,1,Y,1\nSTUDY123,STUDY123-101-9,9,101,54,F,112,162,Treatment 1,VISIT 2,2,12,10.84,10.84,ANALYTE1,ANL1,311.487,,,Y,1,Y,1\n",
```

Comment:
This is one of two largest detours.

1. Instead of pointing to a directory I'm inlining the entire CSV string with the `source` attribute. I could change `source` to `data`. That may make more sense.
2. I removed the `name` attribute.

Reply:
I worry about serialization problems here. Both sides must use the same serialization scheme. You may want to consider Base-64.

Reply:
Please see the [CloudEvents Spec](https://github.com/cloudevents/spec/blob/v1.0.1/spec.md). The source is a reference to the context of the event, the data should go into the "data" field.

Reply:
Moved `source` to `data`. I updated my code in `projectx` and the sample here.